README.md

```diff
@@ -80,16 +88,35 @@ cargo test
 # 
 # test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
 ```
-7. Think about problems you want to solve, sketch them, and consider solving another exercise in this repo (go back to item 3)
-8. If you've finished working on `georust` exercises, think about new challenges you'd like to add and how to contribute to the community
+7. In the terminal navigate back to the root directory of `georustlings` and try another exercise
+8. Think about problems you want to solve, sketch them, and consider solving another exercise in this repo (go back to item 3)
+9. If you've finished working on `georust` exercises, think about new challenges you'd like to add and how to contribute to the community
 
 <!-- Todo: add a nice schematic diagram of this? -->
 
+## Hints and tips
+
+There are many excellent resources for learning Rust online, notably the Rust book and the `rustlings` gamified exercises that inspired this repo.
+Many of those resources refrain from giving advice down to the level of on which editors to use and how to set-up a productive environment for working with Rust.
```

Comment:
I indeed don't often see this level of advice, but I agree it ought to be quite helpful to some people! And anyone else can just ignore it